src/nested_artboard.cpp

```diff
@@ -28,18 +28,29 @@ void NestedArtboard::nest(Artboard* artboard) {
     m_NestedInstance->advance(0.0f);
 }
 
+static Mat2D makeTranslate(const Artboard* artboard) {
+    return Mat2D::fromTranslate(-artboard->originX() * artboard->width(),
+                                -artboard->originY() * artboard->height());
+}
+
 void NestedArtboard::draw(Renderer* renderer) {
     if (m_NestedInstance == nullptr) {
         return;
     }
     renderer->save();
-    renderer->transform(worldTransform());
-    renderer->translate(-m_NestedInstance->originX() * m_NestedInstance->width(),
-                        -m_NestedInstance->originY() * m_NestedInstance->height());
+    renderer->transform(worldTransform() * makeTranslate(m_NestedInstance));
```

Comment:
Now that we have the frameOrigin option, we may be able to just set that on the instance and not need to do this transformation here. I need to try it, fine for now!